P0009/P0009.md

```diff
@@ -1355,28 +1357,37 @@ of rank equal to `sizeof...(Extents)`.
 
 [2]{.pnum} `extents<Extents...>` is a trivially copyable type.
 
-[3]{.pnum} $E_r$ is a _dynamic extent_ if it is equal to `dynamic_extent`, otherwise $E_r$ is a _static extent_.
+[3]{.pnum} Let $E_r$ be the $r^{th}$ element of $Extents$.
+
+[4]{.pnum} $E_r$ is a _dynamic extent_ if it is equal to `dynamic_extent`, otherwise $E_r$ is a _static extent_.
 For each $E_r$ equivalent to `dynamic_extent`,  
 the upper bound of the interval is stored in the exposition-only array `dynamic_extents_` at
 `dynamic_extents_[dynamic_index(`_r_`)]`.
 
-[4]{.pnum} If $E_r$ is a dynamic extent, let $D_r$ be the value of `dynamic_extents_[dynamic_index(`_r_`)]`.
+[5]{.pnum} If $E_r$ is a dynamic extent, let $D_r$ be the value of `dynamic_extents_[dynamic_index(`_r_`)]`.
 The $r^{th}$ interval of an `extents` is as follows: 
   
-  * [4.1]{.pnum} $[0, E_r)$ if $E_r$ is a static extent,
-  
-  * [4.2]{.pnum} otherwise $[0, D_r)$. 
+  * [5.1]{.pnum} $[0, E_r)$ if $E_r$ is a static extent,
+
+  * [5.2]{.pnum} otherwise $[0, D_r)$. 
 
 ---
 
 ```c++
 constexpr size_t dynamic_index(size_t i) noexcept; // @_exposition only_@
 ```
 
-* [5]{.pnum} *Returns*: If `i <= sizeof...(Extents)` is `true`, the number of arguments 
-before the `i`_th_ template parameter in the template parameter pack `Extents` 
+* [7]{.pnum} *Returns*: If `i <= sizeof...(Extents)` is `true`, the number of arguments
```

Comment:
```suggestion
* [7]{.pnum} *Returns*: If `i <= sizeof...(Extents)` is `true`, then the number of arguments
```

Reply:
i think that was a general thing that you don't say "then" for the returns statement.